packages/core/src/database/update-where.ts

```diff
@@ -57,7 +76,11 @@ export const buildUpdateWhereWithPool =
         })
         .filter((value): value is Truthy<typeof value> => notFalsy(value));
 
-    return async ({ set, where, jsonbMode }: UpdateWhereData<Schema>) => {
+    return async <SetKey extends Key, WhereKey extends Key>({
```

Comment:
Do we have use cases where the SetKeys generic type and WhereKeys generic type are different? Since they all come from the same Table schema source

Reply:
P.S. Looks like no run-time logic changes. Should this be marked as a breaking change?

Reply:
`BuildUpdateWhere` needs more flexibility to be compatible with the new methods

Reply:
some of the original types are not working. so marking as breaking change

Reply:
Discussed offline.